src/sg/gov/moh/fhir/4.0.1/healtcert.json

```diff
@@ -0,0 +1,232 @@
+            "extension": [
+              {
+                "url": "code",
+                "valueCodeableConcept": {
+                  "text": "SG"
+                }
+              }
+            ]
+          }
+        ],
+        "identifier": [
+          {
+            "type": {
+              "coding": [
+                {
+                  "system": "http://terminology.hl7.org/CodeSystem/v2-0203",
+                  "code": "PPN"
+                }
+              ]
+            },
+            "value": "E7831177G"
+          },
+          {
+            "type": {
+              "coding": [
+                {
+                  "code": "NRIC"
+                }
+              ]
+            },
+            "value": "S9098989Z"
+          }
+        ],
+        "name": [
+          {
+            "text": "Tan Chen Chen"
+          }
+        ],
+        "gender": "female",
+        "birthDate": "1990-01-15"
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:bbbb1321-4af5-424c-a0e1-ed3aab1c349d",
+      "resource": {
+        "resourceType": "Specimen",
+        "type": {
+          "coding": [
+            {
+              "system": "http://snomed.info/sct",
+              "code": "258500001",
+              "display": "Nasopharyngeal swab"
+            }
+          ]
+        },
+        "collection": {
+          "collectedDateTime": "2020-09-27T06:15:00Z"
+        }
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:99991321-4af5-424c-a0e1-ed3aab1c349d",
+      "resource": {
+        "resourceType": "Organization",
+        "name": "MOH",
+        "type": [
+          {
+            "coding": [
+              {
+                "system": "http://terminology.hl7.org/CodeSystem/organization-type",
+                "code": "govt",
+                "display": "Government"
+              }
+            ]
+          }
+        ]
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:11111321-4af5-424c-a0e1-ed3aab1c349d",
+      "resource": {
+        "resourceType": "Practitioner",
+        "name": [
+          {
+            "text": "Dr Michael Lim"
+          }
+        ],
+        "qualification": [
+          {
+            "identifier": [
+              {
+                "system": "http://example.org/UniversityIdentifier",
+                "value": "MCR 123214"
+              }
+            ],
+            "code": {
+              "coding": [
+                {
+                  "system": "http://terminology.hl7.org/CodeSystem/v2-0360/2.7",
+                  "code": "BS",
+                  "display": "Bachelor of Science"
+                }
+              ],
+              "text": "Bachelor of Science"
+            },
+            "issuer": {
+              "reference": "urn:uuid:99991321-4af5-424c-a0e1-ed3aab1c349d"
+            }
+          }
+        ]
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:cccc1321-4af5-424c-a0e1-ed3aab1c349d",
+      "resource": {
```

Comment:
In order to tie a specimen to an Observation would we need to include a specimen reference? 
Like: 
`"specimen": 
          {
            "reference": "urn:uuid:bbbb1321-4af5-424c-a0e1-ed3aab1c349d"
          },`

Reply:
It's the other way around. An observation can reference a specimen (explanation is in the main message)

A specimen can reference a patient

Reply:
Ahh okay, so just to check in this sample JSON in order to reference a Sample in the Observation resource we should add a reference like this?

`"specimen": {
"reference": "urn:uuid:bbbb1321-4af5-424c-a0e1-ed3aab1c349d"
},`

Reply:
yes, however be careful because you refer here to the fhir schema version. I added the same support on the current used healthcerts schema.

Example: https://raw.githubusercontent.com/Open-Attestation/schemata/feat/fhir/src/sg/gov/moh/healthcert/1.0/sample-document-with-references.json

Reply:
Got it! Thanks!

Would both schemas be accepted as HealthCerts? Or would we need to use the updated HealthCerts schema?

Reply:
we will support first the updated version of the current schema, because it involves less work for providers.

imho the fhir should be supported in the future and be used as reference, but I dont decide on the timeline